src/chatlog/content/filetransferwidget.cpp

```diff
@@ -537,3 +539,29 @@ void FileTransferWidget::onPreviewButtonClicked()
 {
     handleButton(ui->previewButton);
 }
+
+QPixmap FileTransferWidget::scaleCropIntoSquare(const QPixmap &source, const int targetSize) {
```

Comment:
https://github.com/tux3/qTox/wiki/Coding#coding-style

i.e. should be
````
function()
{
    something;
}
```

~Same goes for the rest of code.